### PR TITLE
Metrics for members-data-api endpoints

### PR DESCRIPTION
**Pull request by pawel-krupinski-guardian**
Count and duration for all calls to members-data-api endpoints.
Added so we have more information about which endpoints are being used and how often (and how fast they are).

### CODE REVIEW

File: membership-attribute-service/app/controllers/AttributeController.scala [hunk 1]
```diff
@@ -96,13 +96,14 @@ class AttributeController(
...
       requiredScopes: List[AccessScope],
+      metricName: String,
   ) = {
     AuthAndBackendViaAuthLibAction(requiredScopes).async { implicit request =>
       if (endpointDescription == "membership" || endpointDescription == "features") {
         DeprecatedRequestLogger.logDeprecatedRequest(request)
       }
 
-      request.user match {
+      metrics.measureDuration(metricName)(request.user match {
```
Comment (rupertbates) [inline membership-attribute-service/app/controllers/AttributeController.scala:106]: Does this give us number of requests as well as the duration of the requests?

Comment (pawel-krupinski-guardian) [reply membership-attribute-service/app/controllers/AttributeController.scala:106]: Yes, it sends 2 metrics.

File: membership-attribute-service/app/controllers/AccountControllerMetrics.scala [hunk 2]
```diff
@@ -1,11 +0,0 @@
-package controllers
-
-import com.amazonaws.regions.{Region, Regions}
-import com.gu.monitoring.CloudWatch
-
-class AccountControllerMetrics(val stage: String) extends CloudWatch {
```
Comment (rupertbates) [inline membership-attribute-service/app/controllers/AccountControllerMetrics.scala:6]: Was this not being used?

File: membership-attribute-service/app/controllers/AttributeController.scala [hunk 3]
```diff
@@ -35,8 +36,7 @@ class AttributeController(
     with LoggingWithLogstashFields {
   import commonActions._
   implicit val executionContext: ExecutionContext = controllerComponents.executionContext
-  lazy val metrics = Metrics("AttributesController", stage.value)
-  lazy val expensiveMetrics = new ExpensiveMetrics("AttributesController", stage.value)
+  lazy val metrics = Metrics(getClass.getSimpleName, stage.value)
```
Comment (rupertbates) [inline membership-attribute-service/app/controllers/AttributeController.scala:39]: 🥇

File: membership-attribute-service/test/controllers/AttributeControllerTest.scala [hunk 4]
```diff
@@ -146,7 +146,7 @@ class AttributeControllerTest extends Specification with AfterAll with Mockito {
       FakePostgresService(validUserId),
       FakeMobileSubscriptionService,
       addGuIdentityHeaders,
-      Stage("PROD"),
+      Stage("LOCAL"),
```
Comment (rupertbates) [inline membership-attribute-service/test/controllers/AttributeControllerTest.scala:149]: This should be `DEV` I think

**Review by rupertbates — APPROVED** [review]
Looks good, just change the stage in the test to DEV and then 👍

**Pull request merged**